feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/tunnel_interface_based_resize_test.go

```diff
@@ -0,0 +1,1572 @@
+			ipAddr:   p.r0Fti0Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti0Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+
+		{
+			desc:     "tunnel-2",
+			intfName: "fti1",
+			ipAddr:   p.r0Fti1Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti1Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+
+		{
+			desc:     "tunnel-3",
+			intfName: "fti2",
+			ipAddr:   p.r0Fti2Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti2Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+		{
+			desc:     "tunnel-4",
+			intfName: "fti3",
+			ipAddr:   p.r0Fti3Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti3Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+
+		{
+			desc:     "tunnel-5",
+			intfName: "fti4",
+			ipAddr:   p.r0Fti4Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti4Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+
+		{
+			desc:     "tunnel-6",
+			intfName: "fti5",
+			ipAddr:   p.r0Fti5Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti5Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+		{
+			desc:     "tunnel-7",
+			intfName: "fti6",
+			ipAddr:   p.r0Fti6Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti6Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+		{
+			desc:     "tunnel-8",
+			intfName: "fti7",
+			ipAddr:   p.r0Fti7Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r0Fti7Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+	}
+
+	// Configure the interfaces.
+	for _, intf := range dutIntfs {
+		t.Logf("Configure DUT interface %s with attributes %v", intf.intfName, intf)
+		i := &oc.Interface{
+			Name:        ygot.String(intf.intfName),
+			Description: ygot.String(intf.desc),
+			Type:        oc.IETFInterfaces_InterfaceType_ethernetCsmacd,
+			Enabled:     ygot.Bool(true),
+		}
+		// configure ipv4 address
+		i.GetOrCreateEthernet()
+		i4 := i.GetOrCreateSubinterface(0).GetOrCreateIpv4()
+		a := i4.GetOrCreateAddress(intf.ipAddr)
+		a.PrefixLength = ygot.Uint8(intf.ipv4mask)
+
+		// configure ipv6 address
+		i6 := i.GetOrCreateSubinterface(0).GetOrCreateIpv6()
+		b := i6.GetOrCreateAddress(intf.ipv6Addr)
+		b.PrefixLength = ygot.Uint8(intf.ipv6mask)
+		gnmi.Replace(t, dut, gnmi.OC().Interface(intf.intfName).Config(), i)
+	}
+}
+
+func ConfigureTunnelDecapDUT(t *testing.T, p *parameters, dut *ondatra.DUTDevice, dp1 *ondatra.Port, dp2 *ondatra.Port, dp3 *ondatra.Port, dp4 *ondatra.Port) {
+
+	dutIntfs := []struct {
+		desc     string
+		intfName string
+		ipAddr   string
+		ipv6Addr string
+		ipv4mask uint8
+		ipv6mask uint8
+	}{
+		{
+			desc:     "R1_ATE1",
+			intfName: dp1.Name(),
+			ipAddr:   p.r1Intf3Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r1Intf3Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		}, {
+			desc:     "R1_ATE",
+			intfName: dp2.Name(),
+			ipAddr:   p.r1Intf4Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r1Intf4Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		}, {
+			desc:     "R1_R0_1",
+			intfName: dp3.Name(),
+			ipAddr:   p.r1Intf5Ipv4Add,
+			ipv4mask: p.ipv4Mask,
+			ipv6Addr: p.r1Intf5Ipv6Add,
+			ipv6mask: p.ipv6Mask,
+		},
+		{
```

Comment:
inconsistent brace styling. On prior lines the opening brace was on the same line as the previous closing one.